style.css

```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins&display=swap');
+
+body{
```

Comment:
Please make sure to leave a space after the selector before the opening curly brackets. Kindly check for the rest of the CSS selectors.